### PR TITLE
zh:change cluster-version

### PR DESCRIPTION
**Pull request by liangyuanpeng**
change_cluster_version

### CODE REVIEW

**Comment by rootsongjc** [comment]
Please sign the CLA first.

**Comment by liangyuanpeng** [comment]
should I use the github account of this email to submit @rootsongjc

**Comment by rootsongjc** [comment]
@liangyuanpeng CI failed but not your fault, you can merge the upstream and commit again to trigger to CI.

File: content_zh/docs/setup/kubernetes/download-release/index.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
+    ---
```
Comment (rootsongjc) [inline content_zh/docs/setup/kubernetes/download-release/index.md]: Please delete the black spaces in front of `---`.

File: .spelling [hunk 2]
```diff
@@ -107,6 +107,7 @@ Kube
 Kubecon
 Kubelet
 Kubernetes
+kubernetes
```
Comment (rootsongjc) [inline .spelling]: The initial of `Kubernetes` should be capitalized. No need the add this spelling.

**Comment by rootsongjc** [comment]
This PR has no change with the original files, so close it.

**Pull request closed without merging**